scripts/inject-container-reference.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+sha_reference="$1"
+now="$(date --rfc-3339=seconds)"
+csvfile="bundle/manifests/azure-service-operator.clusterserviceversion.yaml"
```

Comment:
This isn't a csvfile?

Reply:
It's a Cluster Service Version file. :grin: I blame the Openshift people for picking an overloaded acronym! But I'll rename the variable to something less confusing.

Reply:
haha I totally missed that!